allenact/setup.py

```diff
@@ -3,20 +3,61 @@
 
 from setuptools import find_packages, setup
 
+
+def parse_req_file(fname, start=None):
+    reqs = {} if start is None else start
+    cline = None
+    with open(fname, "r") as f:
+        for line in f.readlines():
+            line = line[:-1]
+            if len(line) == 0:
+                continue
+            if line[0] == "[":
+                cline = line[1:-1]
+                reqs[cline] = []
+            else:
+                if cline is not None:
+                    reqs[cline].append(line)
+    return reqs
+
+
+def get_version(fname):
+    res = "UNK"
+    with open(fname, "r") as f:
+        for line in f.readlines():
+            line = line[:-1]
+            if line.startswith("Version:"):
+                res = line.replace("Version:", "").strip()
+                break
+    return res
+
+
 if __name__ == "__main__":
-    os.chdir(os.path.join(os.path.abspath(os.path.dirname(Path(__file__))), ".."))
+    base_dir = os.path.abspath(os.path.dirname(Path(__file__)))
+
+    if not os.path.exists(
+        os.path.join(base_dir, "allenact.egg-info/dependency_links.txt")
```

Comment:
Will this actually be built under the `allenact/allenact` dir? I thought it would be under the top level dir?

Reply:
when we generate the wheel from the sdist, the top level os the `base_dir`

Reply:
(unless I messed up the mess, if you know what I mean :D)